HabitRPG/HRPGTopHeaderNavigationController.m

```diff
@@ -63,11 +62,11 @@ - (void)viewWillLayoutSubviews {
     [super viewWillLayoutSubviews];
     CGRect parentFrame = self.view.frame;
     self.backgroundView.frame =
-        CGRectMake(0, self.headerYPosition, parentFrame.size.width, self.topHeaderHeight+6);
+        CGRectMake(0, self.headerYPosition, parentFrame.size.width, [self topHeaderHeight]+6);
```

Comment:
Good question! I actually removed the property as a whole (see [line 20](https://github.com/schrockblock/habitica-ios/pull/1/files#diff-3cf0a5a938239ec1895e2965bcb54d97L20) above). `topHeaderHeight` is never set, and is instead only ever calculated (see [here](https://github.com/schrockblock/habitica-ios/pull/1/files#diff-3cf0a5a938239ec1895e2965bcb54d97R170)), so I figured it was more of a method than a property. But happy to change it back if you prefer!